tests/scripts/kubeadm-install.sh

```diff
@@ -4,7 +4,7 @@ KUBE_VERSION=${1:-"v1.7.5"}
 
 null_str=
 KUBE_INSTALL_VERSION="${KUBE_VERSION/v/$null_str}"-00
-
+sudo swapoff -a
```

Comment:
why do we need to disable swap? a comment would be good here

Reply:
yes for K8s 1.8 we need to disable swap. I will put a comment in.